community-tutorials/debootstrap/01-en.md

```diff
@@ -0,0 +1,311 @@
+
+```
+|start|[sda1 - grub][sda2 - swap][sda3 - ext4 - mounted as / - ]|end|
+```
+
+This setup is quite disadvantageous since we are not able to shrink an ext4 filesystem while it is mounted. With Netcup we are lucky since inside the Server Control Panel, we can boot from an iso **and** have a console with mouse support.
+
+*For this step we will have some downtime. For my VPS 16years (with an HDD) downsizing the ext4 by around 500 GB took about 50 minutes.*
+
+## Step 1.1 - (Shrink existing partition)
+![Gparted DVD in Netcup SCP](images/netcup_scp.png)
+
+Log into your SCP, select your server, then go to Media and select DVD Drive. There we insert the GParted Live CD and set the server to boot from DVD via switch. Afterwards we reboot the server and open the console via the >_Screen menupoint.
+After a short time you should see the gparted boot prompt. You can take the time to reconfigure the setup (e.g. if you want german keyboard) but you should be okay with the default settings (press enter 4 times). After that open gparted. Select the partition, click move/resize and resize the partition to the desired size. The free space should be at least large enough to house a new partition. 
+
+## Step 1.2 - (Create partition for the new system)
+![Add partition in gparted](images/gparted.png)
+
+Now create a new partition inside the new space and format it to ext4. Other formats like xfs or btrfs would be possible as well but we stick to the standard for this tutorial. 
+
+## Step 1.3 - (Reboot into linux)
+With the new partition we can now reboot into our existing system. For that first close the gparted gui and then doubleclick the red shutdown icon. Select reboot and confirm. You should then see a prompt to remove the gparted live cd.
+Disable the boot to DVD from the scp and restart your machine with the [detach DVD] button.
+
+Checkout the name of your new partition using `lsblk`. If your setup was exactly like described earlier, it should be /dev/sda4. In the following steps we will call this drive /dev/sdaX.
+
+## Alternate option with less downtime
+
+If your system contains a large enough swap partition (e.g. 4G), it is possible to reuse this part of the harddrive as a new system. In this case you would disable the swap partition in the running system `swapoff -a`, delete it and recreate it as an ext4 to house the new system. Make sure that your system has enough ram to run without swap. Also make sure that you remove the swap entry in your `/etc/fstab` to avoid a long waiting time while the system is looking for the swap partition at reboot. After booting to the new system consider resizing your old system and recreate a swap partition elsewhere.
+
+# Step 2 - (Install your new base system)
+
+Create a mountpoint for your new partition:
+```
+mkdir /mnt/debinst
+```
+Mount the newly created drive via
+```
+mount /dev/sdaX
```

Comment:
I think you forgot to specify the mountpoint.

Reply:
I've added the mount point